lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -1621,6 +1626,10 @@ private function validateAndCompleteTypedFieldMapping(array $mapping): array
                     case 'string':
                         $mapping['type'] = Types::STRING;
                         break;
+                    default:
```

Comment:
Is the way we want it? You couldn't for example override string globally to map to a different Type with this implementation. I would think we can improve this by moving all the default type mappings into `typedFieldMappings` as default value.

Reply:
ok, good idea, Ill implement this

Reply:
can we wait with 2.14.x for this? id hate to have it postponed until 2.15.x whe its almost done